README.md

```diff
@@ -1,72 +1,103 @@
 # hello-world-fwless
-Simple java rest API without framework
+Simple java API rest without framework
```

Comment:
I propose as text :

- Simple Java HTTP application designed without a framework but instead using the java class `com.sun.net.httpserver.HttpServer`